### PR TITLE
change components name

### PR DESCRIPTION
**Pull request by sweetcuptea**
**What issue/feature/bugfix/improvement did you solve?**
---
Over time, I feel that my components will be a little bit difficult to read because I add a custom name `Tx` on the component
So, I decide to remove it

**How you resolve the issue**
---
I remove `Tx` from all of my components

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div id="app">
-    <tx-preloader v-if="isLoading" :progress="progress" />
+    <preloader v-if="isLoading" :progress="progress" />
```
Comment (trihargianto) [inline src/App.vue:3]: Component name should always be multi-word. This is what Vue team recommended https://v2.vuejs.org/v2/style-guide/?redirect=true#Multi-word-component-names-essential

Comment (trihargianto) [reply src/App.vue:3]: If you want to remove the Tx prefix, please make sure your component using multi-word naming convention

Comment (sweetcuptea) [reply src/App.vue:3]: I decided to add V to my components name as a prefix as it said https://v2.vuejs.org/v2/style-guide/#Base-component-names-strongly-recommended

Comment (trihargianto) [reply src/App.vue:3]: ok

**Review by trihargianto — APPROVED** [review]
Looks daijoubu to me ✅

**Pull request merged**